docs/Makefile

```
@@ -15,12 +15,21 @@ BUILDDIR      = build
 help:
 	@$(SPHINXBUILD) -M help "$(SOURCEDIR)" "$(BUILDDIR)" $(SPHINXOPTS) $(O)
 livehtml:
-	@$(SPHINXAUTOBUILD) -z ../leapp "$(SOURCEDIR)" "$(BUILDDIR)/html" $(SPHINXOPTS) $(0)
+	@$(SPHINXAUTOBUILD) --watch ../leapp "$(SOURCEDIR)" "$(BUILDDIR)/html" $(SPHINXOPTS) $(0)
 
-.PHONY: help Makefile
+# Prepare virtualenv similar to the environment used during build on Read The Docs and install required packages
+env:
+	PY_VER=$$(sed -n -e 's/^\s*python:\s"\(.*\)"/\1/p' ../.readthedocs.yaml); \
+	echo $$PY_VER; \
+	rm -fr ./venv; \
+	virtualenv venv --python=python$$PY_VER; \
+	source venv/bin/activate; \
+	# pip install -r doesn't really work with relative paths :), so keeping it in the project root
+	pushd .. && pip install -r requirements-docs.txt; popd
```

```suggestion
	pushd .. && pip install -r requirements-docs.txt; popd
```

comments must not be in the middle of the command. the comment here actually split the command in to different executions which leads to a pip installation of requrements to user directories (~/.local/....) instead of the `venv` environment.